tests/migrations/test_commands.py

```diff
@@ -695,6 +695,40 @@ def test_sqlmigrate_for_non_transactional_databases(self):
             self.assertNotIn(start_transaction_sql.lower(), queries)
         self.assertNotIn(connection.ops.end_transaction_sql().lower(), queries)
 
+    @override_settings(MIGRATION_MODULES={"migrations": "migrations.test_migrations_squashed"})
+    def test_sqlmigrate_squashed_migration(self):
+        """
+        sqlmigrate squashed migration outputs forward looking SQL.
+        """
+        out = io.StringIO()
+        # ambiguous migration name
+        msg = "More than one migration matches '0001' in app 'migrations'. Please be more specific."
+        with self.assertRaisesMessage(CommandError, msg):
+            call_command("sqlmigrate", "migrations", "0001", stdout=out)
+
+        out = io.StringIO()
+        # assert we find the replaced migration
+        call_command("sqlmigrate", "migrations", "0001_initial", stdout=out)
+        output = out.getvalue().lower()
+        self.assertNotEqual(output.find('-- create model author'), -1)
+        self.assertNotEqual(output.find('create table'), -1)
+        self.assertNotEqual(output.find('-- create model tribble'), -1)
+
+        out = io.StringIO()
+        # assert we find the second replaced migration
+        call_command("sqlmigrate", "migrations", "0002_second", stdout=out)
+        output = out.getvalue().lower()
+        self.assertNotEqual(output.find('-- delete model tribble'), -1)
+        self.assertNotEqual(output.find('-- create model book'), -1)
+
+        out = io.StringIO()
+        # assert can also inspect the SQL of the squashed migration
+        call_command("sqlmigrate", "migrations", "0001_squashed_0002", stdout=out)
+        output = out.getvalue().lower()
+        self.assertNotEqual(output.find('-- create model author'), -1)
+        self.assertEqual(output.find('-- create model tribble'), -1)
+        self.assertNotEqual(output.find('-- create model book'), -1)
```

Comment:
Please split into 4 tests, one for each thing being tested, and migrate those comments into the test names.